### PR TITLE
Pull branch

### PR DESCRIPTION
**Pull request by marco-brandizi**
This fixes the changes discussed in pull request #2.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -124,24 +144,28 @@
...
+      <repository>
+          <id>${distro.id}</id>
+          <url>${distro.url}</url>
+      </repository>
+      
+    </distributionManagement>
+      
     <scm>
-        <url>https://github.com/harti2006/neo4j-server-maven-plugin</url>
+        <url>https://github.com/Rothamsted/neo4j-server-maven-plugin</url>
```
Comment (harti2006) [inline pom.xml]: Only this little change is left, then I will merge everything :-)

**Comment by marco-brandizi** [comment]
Gosh, sorry, hope I haven't other leftovers like that! :-)

**Comment by marco-brandizi** [comment]
Cool! Thanks!

**Comment by harti2006** [comment]
I also released your changes in a new version 1.0.0 and switched to automatic deployments with version 1.0.1, including snapshots as well. Thanks for your contribution.

**Pull request merged**